crates/modelardb_storage/src/delta_lake.rs

```diff
@@ -504,3 +504,159 @@ impl DeltaLake {
         format!("{}/{METADATA_FOLDER}/{table_name}", self.location)
     }
 }
+
+/// Functionality for transactionally writing [`RecordBatches`](RecordBatch) to a Delta Table stored
+/// in a local folder or an object store.
```

Comment:
Maybe just say `stored in an object store` since local folders are also object stores.

Reply:
Fixed.